presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -290,6 +291,14 @@ private DirContext createUserDirContext(String userDistinguishedName, String pas
                 .build();
     }
 
+    private static void validateLdapUrl(String ldapUrl, boolean allowInsecure)
+    {
+        if (ldapUrl.matches("ldap://.*")) {
+            checkState(allowInsecure, "Connecting to the LDAP server without SSL enabled requires `ldap.allow-insecure=true`");
+            log.warn("Presto will use non-secure connection to the LDAP server. Please consider using an SSL connection for improved security.");
```

Comment:
It's better to tell users about the specific threat rather than a vague "improved security". How about

> Passwords will be sent in the clear to the LDAP server. Please consider using SSL to connect.